### DISCUSSION TITLE
DatePicker - Is it possible to validation error message ?

### DISCUSSION THREAD

**Discussion by makarandkeer** [adobe/react-spectrum#3305] [General]
For DatePicker, if user enters any invalid date / time, currently it shows `error` icon, however from user point of view its difficult to know what exactly is wrong. 
Is it possible to show error message/text along with `error` icon.

For reference here is how it shows currently and as you can notice; without proper error-text user will not know whats wrong in this date/time
![image](https://user-images.githubusercontent.com/32228809/178774177-0126b784-3491-4e85-b565-405e11bd2843.png)

**Comment by devongovett** [comment]
The `errorMessage` prop should do what you want: https://react-spectrum.adobe.com/react-spectrum/DatePicker.html#help-text

**Reply by makarandkeer** [reply]
Thanks for suggestion. `errorMessage` when set does show up but not inside Calendar pop-up (please see ss below) 
![image](https://user-images.githubusercontent.com/32228809/178850674-8b06c4a3-a018-41e8-aebf-fa2d370acc0a.png)

**Reply by devongovett** [reply]
hmm it works for me

<img width="449" alt="image" src="https://user-images.githubusercontent.com/19409/178852483-b05ba8ac-605c-44a2-a287-47b98e298f12.png">

**Reply by makarandkeer** [reply]
Can you try by setting `granularity="minute"` ?

**Reply by makarandkeer** [reply]
Can you please let me know if issue is reproducible when granularity is set (for example granularity="minute")?

**Reply by snowystinger** [reply]
It might be easiest if you tried reproducing in a codesandbox so we can see it as well.
https://codesandbox.io/s/react-spectrum-template-syueo
